mypy.ini

```
@@ -0,0 +1,6 @@
+[mypy]
+warn_unused_configs = True
+files = securesystemslib/util.py
+# Supress error messages until enough modules
+# are type annotated
+follow_imports = silent
```

```suggestion
follow_imports = silent

```
Very much a nit, but tools will keep complaining about not having a newline at the end of the file.